static/js/click_handlers.js

```diff
@@ -505,7 +505,14 @@ exports.initialize = function () {
         do_render_buddy_list_tooltip(elem, title_data);
     });
 
-    $('#user_presences').on('mouseleave click', '.user-presence-link, .user_sidebar_entry .user_circle, .user_sidebar_entry .selectable_sidebar_block', function (e) {
+    $('#user_presences').on('click', '.user-presence-link, .user_sidebar_entry .user_circle, .user_sidebar_entry .selectable_sidebar_block', function (e) {
```

Comment:
This could become an endless list if UI elements keep getting added. Why don't you try the opposite approach. Look for *any* click event outside the dropdown-menu and close the menu if when this event is triggered,